export/csv/wormhole.csv

```diff
@@ -1,81 +1,81 @@
-"31";"L005";"C2";;"5000000";"1";"16";
-"32";"L477";"C3";"2000000000";"300000000";;"16";
-"33";"L614";"C5";"1000000000";"20000000";;"24";
-"34";"M006";"C4";;"5000000";"1";"16";
-"35";"M267";"C3";"1000000000";"300000000";;"16";
-"36";"M555";"C5";"3000000000";"1000000000";;"24";
-"37";"M609";"C4";"1000000000";"20000000";;"16";
-"38";"N062";"C5";"3000000000";"300000000";;"24";
-"39";"N110";"H";"1000000000";"20000000";;"24";
-"40";"N290";"L";"5000000000";"1800000000";"500000000";"24";
-"41";"N432";"C5";"3000000000";"1350000000";;"24";
-"42";"N766";"C2";"2000000000";"300000000";;"16";
-"43";"N770";"C5";"3000000000";"300000000";;"24";
-"44";"N944";"L";"3000000000";"1350000000";;"24";
-"45";"N968";"C3";"2000000000";"300000000";;"16";
-"46";"O128";"C4";"1000000000";"300000000";"100000000";"24";
-"47";"O477";"C3";"2000000000";"300000000";;"16";
-"48";"O883";"C3";"1000000000";"20000000";;"16";
-"49";"P060";"C1";"500000000";"20000000";;"16";
-"50";"Q003";"0.0";;"5000000";"1";"16";
-"51";"Q317";"C1";"500000000";"20000000";;"16";
-"52";"R051";"L";"3000000000";"1000000000";;"16";
-"53";"R474";"C6";"3000000000";"300000000";;"24";
-"54";"R943";"C2";"750000000";"300000000";;"16";
-"55";"S047";"H";"3000000000";"300000000";;"24";
-"56";"S199";"0.0";"3000000000";"1350000000";;"24";
-"57";"S804";"C6";"1000000000";"20000000";;"24";
-"58";"T405";"C4";"2000000000";"300000000";;"16";
-"59";"U210";"L";"3000000000";"300000000";;"24";
-"60";"U319";"C6";"3000000000";"1800000000";"500000000";"48";
-"61";"U574";"C6";"3000000000";"300000000";;"24";
-"62";"V283";"0.0";"3000000000";"1000000000";;"24";
-"63";"V301";"C1";"500000000";"20000000";;"16";
-"64";"V753";"C6";"3000000000";"1350000000";;"24";
-"65";"V911";"C5";"3000000000";"1350000000";;"24";
-"66";"W237";"C6";"3000000000";"1350000000";;"24";
-"67";"X702";"C3";"1000000000";"300000000";;"24";
-"68";"X877";"C4";"2000000000";"300000000";;"16";
-"69";"Y683";"C4";"2000000000";"300000000";;"16";
-"70";"Y790";"C1";"500000000";"20000000";;"16";
-"71";"Z006";"C3";;"5000000";"1";"16";
-"72";"Z060";"0.0";"1000000000";"20000000";;"24";
-"73";"Z142";"0.0";"3000000000";"1350000000";;"24";
-"74";"Z457";"C4";"2000000000";"300000000";;"16";
-"75";"Z647";"C1";"500000000";"20000000";;"16";
-"76";"Z971";"C1";"100000000";"20000000";;"16";
-"80";"M001";"C4";;"5000000";"1";"16";
-"81";"E587";"0.0";"3000000000";"1000000000";;"16";
-"82";"V898";"L";"2000000000";"300000000";;"16";
-"83";"Q063";"H";"500000000";"20000000";;"16";
\ No newline at end of file
+"Id";"Name";"Security";"MassTotal";"MassIndividual";"MassRegeneration";"MaxStableTime";"SignatureStrength";
+"1";"A009";"L";"500000000";"20000000";;"16";;
+"2";"A239";"L";"2000000000";"300000000";;"24";"5";
+"3";"A641";"H";"2000000000";"1000000000";;"16";"10";
+"4";"A982";"C6";"3000000000";"300000000";;"24";"2.22";
+"5";"B041";"C6";"3000000000";"300000000";"500000000";"48";;
+"6";"B274";"H";"2000000000";"300000000";;"24";"10";
+"7";"B449";"H";"2000000000";"1000000000";;"16";;
```

Comment:
According to http://wiki.eve-inspiracy.com/index.php?title=Wormhole_Signature_Strength_List **B449** has a strength of 2.5%